### PR TITLE
Updating Acquisitions report documentation

### PR DESCRIPTION
**Pull request by Alan1Mulhall**
Adding new Channels & Conversions capabilities that were recently added to the report

### CODE REVIEW

**Comment by Lauragra** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md) | :warning:Warning | | [Details](#user-content-docs%2Fview-acquisitions-report.md)

<a name="docs%2Fview-acquisitions-report.md"></a>
### [docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md)
 - **Line 74**: **[Warning]** The content part of NOTE/WARNING/CAUTION/IMPORTANT/NEXT syntax is suggested to start in a new line.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C4%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201810041710249919-156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=L7oSkfvdh1BlLleZHeNdGHmebR%2Bxn8YSXC3IEcnwv5U%3D&st=2018-10-04T17%3A06%3A48Z&se=2018-11-04T17%3A11%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Lauragra** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C4%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201810041713456290-156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=U36cAn%2BjX3enaAqYOznED%2BpSWzAbHafdlUCllsWNp8o%3D&st=2018-10-04T17%3A09%3A59Z&se=2018-11-04T17%3A14%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Lauragra** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C5%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201810052053067126-156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0gnDh%2FMXLaayYA%2BLOALPu3PIJNAt04R9LvjCJ4vQp2o%3D&st=2018-10-05T20%3A49%3A24Z&se=2018-11-05T20%3A54%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/view-acquisitions-report.md [hunk 1]
```diff
@@ -54,7 +53,38 @@ The values supported within the Acquisitions chart include:
 The **Markets** chart shows the total number of acquisitions over the selected period of time for each market in which your solution is available.
 
 You can view this data in a visual Map form, or toggle the setting to view it in Table form. Table form will show five markets at a time, sorted either alphabetically or by highest/lowest number of acquisitions. You can also download the data to view info for all markets together.
- 
+
+## App page views and conversions by channel
+
+The Acquisition funnel shows you how many customers completed each step of the funnel, from viewing the Store page to using the add-in or app, along with the conversion rate. This data can help you identify areas where you might want to invest more to increase your page views, acquisitions, or activations.
```
Comment (Lauragra) [inline docs/view-acquisitions-report.md]: Okay to refer to the Store page, or should that be the AppSource listing?

**Comment by Lauragra** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C6%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201810061832292934-156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2P%2BNgQB3OaaojCj5clfF9XOP6TU5IThxDU4TPFw4jbc%3D&st=2018-10-06T18%3A28%3A29Z&se=2018-11-06T18%3A33%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/view-acquisitions-report.md (hunk 1 above)
Comment (Alan1Mulhall) [reply docs/view-acquisitions-report.md]: Good call - I'll update!

**Comment by Lauragra** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/view-acquisitions-report.md](https://github.com/Alan1Mulhall/office-store-docs/blob/patch-6/docs/view-acquisitions-report.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C10%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201810100858431906-156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=aQYQokf%2F35%2F1DYfneLZHAaDVePASQpZwP6oFvVTcbj8%3D&st=2018-10-10T08%3A54%3A44Z&se=2018-11-10T08%3A59%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**